CalendarApp/GoogleCalendarService/Extensions/EventCoverExtension.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Google.Apis.Calendar.v3.Data;
+
+namespace GoogleCalendarService.Extensions
+{
+    public static class EventCoverExtension
+    {
+        public static bool LeftCoverConflict(this Event conflictedEvent,Event eventForAdd)
```

Comment:
this method have to be case agnostic so you should not call parameters like conflicted and eventToAdd. It should be smth agnostic like source and target...